README.md

```diff
@@ -28,137 +28,33 @@
-- When you are coding use the [Coding Guidelines](docs/coding-guidelines.md).
-- Please read [How to Contribute](docs/how-to-contribute.md) to understand the procedure and guidelines of making contributions.
-- We have periodically community sync ups in multiple timezones and languages, please find [Community Syncup](docs/community-sync-up.md) to attend online sync ups. 
-
-## Other Resources
-
-**Demo videos**
-
-- Subscribe to [YuniKorn Youtube Channel](https://www.youtube.com/channel/UCDSJ2z-lEZcjdK27tTj_hGw) to get notification about new demos!
-- [Running YuniKorn on Kubernetes - a 12 minutes Hello-world demo](https://www.youtube.com/watch?v=cCHVFkbHIzo)
-- [YuniKorn configuration hot-refresh introduction](https://www.youtube.com/watch?v=3WOaxoPogDY)
-- [Yunikorn scheduling and volumes on Kubernetes](https://www.youtube.com/watch?v=XDrjOkMp3k4)
-- [Yunikorn placement rules for applications](https://www.youtube.com/watch?v=DfhJLMjaFH0)
-
-**Communication channels**
-
-- Mailing lists are:
-  - for people wanting to contribute to or discuss the project development: [dev@yunikorn.apache.org](mailto:dev@yunikorn.apache.org)   
-    subscribe to: dev@ [subscribe](mailto:dev-subscribe@yunikorn.apache.org?subject="subscribe%20to%20YuniKorn%20dev%20list") ([unsubscribe](mailto:dev-unsubscribe@yunikorn.apache.org?subject="unsubscribe%20from%20YuniKorn%20dev%20list") [archives](https://lists.apache.org/list.html?dev@yunikorn.apache.org))
-  - for JIRA issue updates subscribe to: issues@ [subscribe](mailto:issues-subscribe@yunikorn.apache.org?subject="subscribe%20to%20YuniKorn%20issues%20list") ([unsubscribe](mailto:issues-unsubscribe@yunikorn.apache.org?subject="unsubscribe%20from%20YuniKorn%20issues%20list") [archives](https://lists.apache.org/list.html?issues@yunikorn.apache.org))
-  - for GitHub pull requests messages subscribe to: reviews@ [subscribe](mailto:reviews-subscribe@yunikorn.apache.org?subject="subscribe%20to%20YuniKorn%20reviews%20list") ([unsubscribe](mailto:reviews-unsubscribe@yunikorn.apache.org?subject="unsubscribe%20from%20YuniKorn%20reviews%20list") [archives](https://lists.apache.org/list.html?reviews@yunikorn.apache.org))
-
-- We use [Slack](https://slack.com/) as our collaboration system, you can join us by accessing [this link](https://join.slack.com/t/yunikornworkspace/shared_invite/enQtNzAzMjY0OTI4MjYzLTBmMDdkYTAwNDMwNTE3NWVjZWE1OTczMWE4NDI2Yzg3MmEyZjUyYTZlMDE5M2U4ZjZhNmYyNGFmYjY4ZGYyMGE).
-Currently, we have following channels in the workspace: `#yunikorn-dev` and `#yunikorn-user`.
-
-**Blog posts**
-
-Apache based blogs:
-- [Apache YuniKorn (Incubating) release announcements](https://blogs.apache.org/yunikorn/)
-
-3rd party blog posts:
-- [YuniKorn: a universal resource scheduler](https://blog.cloudera.com/blog/2019/07/yunikorn-a-universal-resource-scheduler/)
+The `yunikorn-core` is the brain of the scheduler, which makes placement decisions (Allocate container X on node Y) according
```

Comment:
Shouldn't we mention here the yunikorn-release repository as well? Mainly because of the helm chart.

Reply:
Thanks @kingamarton  updated the doc to include the release and the site repo.